L1Trigger/CSCTriggerPrimitives/python/cscTriggerPrimitiveDigis_cfi.py

```diff
@@ -19,526 +24,68 @@
-    buildLCTfromCLCTandGEM_ME1a = cms.bool(False),
-    buildLCTfromCLCTandGEM_ME1b = cms.bool(True),
-    doLCTGhostBustingWithGEMs = cms.bool(False),
-    promoteALCTGEMpattern = cms.bool(True),
-    promoteALCTGEMquality = cms.bool(True),
-    promoteCLCTGEMquality_ME1a = cms.bool(True),
-    promoteCLCTGEMquality_ME1b = cms.bool(True),
-
-    ## bits for high-multiplicity triggers
-    useHighMultiplicityBits = cms.bool(False),
-)
-
-# to be used by ME21 chambers with GEM-CSC ILT
-me21tmbSLHCGEM = cms.PSet(
-    mpcBlockMe1a    = cms.uint32(0),
-    alctTrigEnable  = cms.uint32(0),
-    clctTrigEnable  = cms.uint32(0),
-    matchTrigEnable = cms.uint32(1),
-    matchTrigWindowSize = cms.uint32(3),
-    tmbL1aWindowSize = cms.uint32(7),
-    verbosity = cms.int32(0),
-    tmbEarlyTbins = cms.int32(4),
-    tmbReadoutEarliest2 = cms.bool(False),
-    tmbDropUsedAlcts = cms.bool(False),
-    clctToAlct = cms.bool(False),
-    tmbDropUsedClcts = cms.bool(False),
-    matchEarliestAlctOnly = cms.bool(False),
-    matchEarliestClctOnly = cms.bool(False),
-    tmbCrossBxAlgorithm = cms.uint32(2),
-    maxLCTs = cms.uint32(2),
-
-    ## run in debug mode
-    debugLUTs = cms.bool(False),
-    debugMatching = cms.bool(False),
-
-    ## use old dataformat
-    useOldLCTDataFormat = cms.bool(True),
-
-    ## matching to pads
-    maxDeltaBXPad = cms.int32(1),
-    maxDeltaBXCoPad = cms.int32(1),
-    maxDeltaPadL1Even = cms.int32(12),
-    maxDeltaPadL1Odd = cms.int32(24),
-    maxDeltaPadL2Even = cms.int32(12),
-    maxDeltaPadL2Odd = cms.int32(24),
-
-    ## efficiency recovery switches
-    dropLowQualityALCTsNoGEMs = cms.bool(True),
-    dropLowQualityCLCTsNoGEMs = cms.bool(True),
-    buildLCTfromALCTandGEM = cms.bool(True),
-    buildLCTfromCLCTandGEM = cms.bool(True),
-    doLCTGhostBustingWithGEMs = cms.bool(False),
-    promoteALCTGEMpattern = cms.bool(True),
-    promoteALCTGEMquality = cms.bool(True),
-    promoteCLCTGEMquality = cms.bool(True),
-
-    ## bits for high-multiplicity triggers
-    useHighMultiplicityBits = cms.bool(False),
-)
-
-# to be used by ME31-ME41 chambers
-meX1tmbSLHC = cms.PSet(
-    mpcBlockMe1a    = cms.uint32(0),
-    alctTrigEnable  = cms.uint32(0),
-    clctTrigEnable  = cms.uint32(0),
-    matchTrigEnable = cms.uint32(1),
-    matchTrigWindowSize = cms.uint32(3),
-    tmbL1aWindowSize = cms.uint32(7),
-    verbosity = cms.int32(0),
-    tmbEarlyTbins = cms.int32(4),
-    tmbReadoutEarliest2 = cms.bool(False),
-    tmbDropUsedAlcts = cms.bool(False),
-    clctToAlct = cms.bool(False),
-    tmbDropUsedClcts = cms.bool(False),
-    matchEarliestAlctOnly = cms.bool(False),
-    matchEarliestClctOnly = cms.bool(False),
-    tmbCrossBxAlgorithm = cms.uint32(2),
-    maxLCTs = cms.uint32(2),
-
-    ## run in debug mode
-    debugLUTs = cms.bool(False),
-    debugMatching = cms.bool(False),
-
-    ## bits for high-multiplicity triggers
-    useHighMultiplicityBits = cms.bool(False),
+    alctPhase1 = alctPhase1,
```

Comment:
General comment, please always use `clone` when you import a python module. Otherwise, we risk to change the original module (in this case `alctPhase1`) by modifying `cscTriggerPrimitiveDigis`